README.md

```diff
@@ -288,6 +288,19 @@ And the parameters that could be used in the search hash are:
         recurring_payment: Active subscription recurring payment.
         subscription_payment: Subscription fee.
 
+### Auto Pagination
+
+It is also possible to inform if whether or not the result should be automatically paginated to return all payments. By default MercadoPago return only 30 payments on each request and if you setting up `auto_paginate` to true, will be made as many requests as needed to return all payments that matches with your search.
+
+    # Create a hash to search for
+    search_hash = { external_reference: 'OPERATION-ID-1234' }
+
+    # setup the auto_paginate to true
+    mp_client.auto_paginate = true
+
+    # do the desired search
+    search = mp_client.search(search_hash)
```

Comment:
@cleicar Why don't we just have `auto_paginate = true` set by default. I think as a developer I'd like that to be the default behavior. If anyone wants to disable it, they can open an issue in this library and we can consider it in the future?

Reply:
@etagwerker  Users of the library expect this old method to return a limited amount of records, and they are (or rather, should be) using their own pagination technique around it. If we made it the default, there may or may not be bugs in their app code stemming from this library change.

I think that making the default value for auto_paginate true is a good idea for the next major version release, since it's a feature many could benefit from, but not for a minor or patch release.

But I'm open to change it though if you don't this concern makes sense.

Reply:
@cleicar I like the idea of changing the behavior in the next major version release. 👍